controllers/ocirepository_controller.go

```diff
@@ -689,6 +689,7 @@ func (r *OCIRepositoryReconciler) craneOptions(ctx context.Context) []crane.Opti
 		crane.WithContext(ctx),
 		crane.WithUserAgent(oci.UserAgent),
 	}
+	options = append(options, crane.Insecure)
```

Comment:
I think it might be better to make this a configurable flag on the OCIRepository object, as setting insecure seems to downgrade HTTPS to HTTP in certain parts of crane: https://github.com/google/go-containerregistry/blob/a0c4bd256482b8522065d5f6cf966281ef270680/pkg/name/registry.go#L75-L78

Reply:
Like we do for Buckets?

Reply:
I've added the same field as for Buckets, please see my latest commit.

Reply:
Thanks :bow: